.github/workflows/release.yml

```diff
@@ -0,0 +1,36 @@
+name: Release
+
+on:
+  workflow_dispath:
+    input:
+      releaseVersion:
+        description: Release Version
+        required: true
+      developmentVersion:
+        description: Next Development Version
+        required: true
+job:
+  releasing:
+    name: Release
+    runs-on: ubuntu-20.04
+    timeout-minutes: 5
+
+    steps:
+      - name: Checkout Project Code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set up Java for publishing to Maven Central Repository
+        uses: actions/setup-java@v2
+        with:
+          java-version: '11'
+          distribution: 'temurin'
+          server-id: github
```

Comment:
What is that field for? In the documentation it is described as "Value of the `distributionManagement/repository/id` field of the `pom.xml`" but I don't see `distributionManagement` field in Hilla's `pom.xml`

Reply:
right, removed it for now, the `distributionManagement` is configured in another branch. Will first get this manual workflow working, then connect the dots.